web/templates/web/_navigation.html

```diff
@@ -10,47 +11,44 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      
-      {% if user.is_researcher %}
-        <ul class="nav navbar-nav navbar-left">
-          <li class="nav-item active"><a class="navbar-item" href="{% url 'web:home' %}">{% trans "Lookit" %}</a></li>
-          <li class="nav-item"><a class="navbar-item active" href="{% url 'exp:study-list' %}">{% trans "Experimenter" %}</a></li>
-          <li class="divider-vertical"></li>
-        </ul>
-      {% else %}
-        <a class="navbar-brand" href="{% url 'web:home' %}">{% trans "Lookit" %}</a>
-      {% endif %}
-
+      <ul class="nav navbar-nav navbar-left">
+        <li class="nav-item active"><a class="navbar-item" href="{% url 'web:home' %}">{% trans "Lookit" %}</a></li>
+        {% if user.is_researcher %}
+        <li class="nav-item"><a class="navbar-item" href="{% url 'exp:study-list' %}">{% trans "Experimenter" %}</a></li>
+        {% endif %}
+        <li class="divider-vertical"></li>
+      </ul>
     </div>
 
     <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
       <ul class="nav navbar-nav navbar-left">
-        <li><a href="{% url 'web:studies-list' %}">{% trans "Studies" %}</a></li>
-        <li><a href="{% url 'web:faq' %}">{% trans "FAQ" %}</a></li>
-        <li><a href="{% url 'web:scientists' %}">{% trans "The Scientists" %}</a></li>
-        <li><a href="{% url 'web:resources' %}">{% trans "Resources" %}</a></li>
+        {% nav_item request 'web:studies-list' 'Studies' %}
+        {% nav_item request 'web:faq' 'FAQ' %}
+        {% nav_item request 'web:scientists' 'The Scientists' %}
+        {% nav_item request 'web:resources' 'Resources' %}
       </ul>
-      
+
       <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-2">
         <ul class="nav navbar-nav navbar-right">
           {% if user.is_authenticated %}
             {% if user.is_researcher %}
-              <li><a href="{% url 'accounts:manage-account' %}">{% trans "My Account" %}</a></li>
+            {% nav_item request 'accounts:manage-account' 'My Account' %}
             {% else %}
-              <li><a href="{% url 'accounts:manage-account' %}">{% trans "Hi" %} {{ user.nickname }}</a></li>
+            {% nav_item request 'accounts:manage-account' 'Hi '|add:user.nickname %}
```

Comment:
is this translated?

Reply:
No.  I'll need to take some time to look into this.

Reply:
Per our conversation, replace "Hi..." with "My Account"